[2021-02-23 14:57:29,859] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_xcom.puller 2021-02-21T00:00:00+00:00 [queued]>
[2021-02-23 14:57:30,376] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_xcom.puller 2021-02-21T00:00:00+00:00 [queued]>
[2021-02-23 14:57:30,412] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:57:30,431] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-23 14:57:30,439] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:57:30,676] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): puller> on 2021-02-21T00:00:00+00:00
[2021-02-23 14:57:31,614] {standard_task_runner.py:52} INFO - Started process 132 to run task
[2021-02-23 14:57:37,655] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_xcom', 'puller', '2021-02-21T00:00:00+00:00', '--job-id', '37', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_xcom.py', '--cfg-path', '/tmp/tmprwmqfisa', '--error-file', '/tmp/tmph2p_1qwk']
[2021-02-23 14:57:37,725] {standard_task_runner.py:77} INFO - Job 37: Subtask puller
[2021-02-23 14:57:45,520] {logging_mixin.py:104} INFO - Running <TaskInstance: example_xcom.puller 2021-02-21T00:00:00+00:00 [running]> on host 0513e3a4eab6
[2021-02-23 14:57:47,246] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_xcom
AIRFLOW_CTX_TASK_ID=puller
AIRFLOW_CTX_EXECUTION_DATE=2021-02-21T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-21T00:00:00+00:00
[2021-02-23 14:57:47,911] {python.py:118} INFO - Done. Returned value was: None
[2021-02-23 14:57:48,136] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_xcom, task_id=puller, execution_date=20210221T000000, start_date=20210223T145729, end_date=20210223T145748
[2021-02-23 14:57:48,496] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-02-23 14:57:48,839] {local_task_job.py:146} INFO - Task exited with return code 0
